SCRAPERS-LIST.md

```diff
@@ -344,6 +344,7 @@ hothouse.com|GammaEntertainment.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
 hotlegsandfeet.com|DDFNetwork.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
 hottiemoms.com|Hustler.yml|:heavy_check_mark:|:x:|:x:|:x:|CDP|-
 hotwifexxx.com|HotWifeXXX.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
+houseofgord.com|houseofgord.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
```

Comment:
```suggestion
houseofgord.com|houseofgord.yml|:heavy_check_mark:|:x:|:x:|:x:|stash >= v0.4.0-14|-
```
till 0.5 gets out we need to update the `needs` column with the version that has cookie support